src/main/java/http_server/Response.java

```diff
@@ -113,6 +114,10 @@ public void head(byte[] bodyContent, String contentType) {
         setHeader(Headers.contentType, contentType);
     }
 
+    public void successfulDelete() {
```

Comment:
Ultimately, this method is setting an instance variable, and this name does not make that clear. And you already have a public `setStatus` function, so I'm curious why you don't use that when setting the status code for a delete.